### ISSUE TITLE
Display gnomad SV allele count (Hom= Het=)

### ISSUE THREAD

**Issue by stephditroia** [broadinstitute/seqr#2507]
**Is your feature request related to a problem? Please describe.**
Seqr displays gnomad SV allele frequency, but given the small SV dataset it is hard to interpret allele count. 

**Describe the solution you'd like**
We would like the gnomad SV hom and het number displayed similar to This Callset or the gnomad v2 and v3 hom count. 

**Additional context**
<img width="1265" alt="image" src="https://user-images.githubusercontent.com/44381119/156798456-73401a02-6661-4ceb-86e4-4459843e0cc8.png">

**Comment by hanars** [comment]
As requested in https://github.com/broadinstitute/seqr/issues/2593, we should also be showing AC once that is available

**Comment by lynnpais** [comment]
@bpblanken we need to look into this

**Comment by lynnpais** [comment]
@bpblanken HET counts are not showing for SNVs as well for gnomAD4. Do you need a separate ticket to track this? 
<img width="1290" alt="Screenshot 2024-07-22 at 9 02 59 AM" src="https://github.com/user-attachments/assets/d9016a18-b345-49c9-96d9-ec33ae866268">

**Comment by bpblanken** [comment]
@lynnpais 

yeah, I think the snv_indel is a separate issue.  Have we ever displayed Het for gnomadAD?  Looks like we do have Hom count being passed through.  

For SVs, I only see `gnomAD_V2_AF`, `gnomAD_V2_AC`, and `gnomAD_V2_AN` in the VCF, which is not sufficient to display Het.  I'm looking into the WDL to see if this is something I can add during the SV annotation work I am doing.

**Comment by lynnpais** [comment]
We display SNV het counts for ExAC and TopMed, but for gnomAD v4 it is in the hover over. Would be good to be consistent. I think this is a separate browser things, not pipeline, so will submit a separate ticket for Hana.

**Comment by lynnpais** [comment]
Will be added to the new dataset in the pipeline. Needs to be updated in the UI.

**Comment by hanars** [comment]
UI code is live but will not be available until data is relaoded
